<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWidget</class>
 <widget class="QWidget" name="mainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1239</width>
    <height>813</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainWidget</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>481</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="setHeader_pb">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>590</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>setHeader</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="picture_path_edit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>290</y>
     <width>371</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="select_pb">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>290</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="one_click_partition_pb">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>290</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>One-click partition</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>630</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>limit:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>670</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>X1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_x1_edit">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>670</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>670</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>X2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_x2_edit">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>670</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>670</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Y1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_y1_edit">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>670</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>670</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Y2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_y2_edit">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>670</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_limited_pb">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>670</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>limited</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>710</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>YZ</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_yz_edit">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>710</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="input_modify_threshold_pb">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>710</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>modify threshold</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>30</y>
     <width>387</width>
     <height>174</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Y2</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLineEdit" name="height_edit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="y1_edit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>X2</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>X1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="yz_edit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="x2_edit"/>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="X_edit"/>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="y2_edit"/>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="id_edit">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="width_edit"/>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="Y_edit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>YZ</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>width</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>height</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="x1_edit"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Y1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>220</y>
     <width>254</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="add_pb">
      <property name="text">
       <string>add</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="modify_pb">
      <property name="text">
       <string>modify</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delete_pb">
      <property name="text">
       <string>delete</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>518</x>
     <y>360</y>
     <width>701</width>
     <height>120</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>comm</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="comm_edit"/>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3" colspan="2">
     <widget class="QPushButton" name="close_comm_pb">
      <property name="text">
       <string>close comm</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="close_CCD_pb">
      <property name="text">
       <string>close CCD</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QPushButton" name="display_CCD_pb">
      <property name="text">
       <string>display CCD</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QPushButton" name="get_picture_pb">
      <property name="text">
       <string>get picture</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Area</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="Area_edit"/>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="3" colspan="2">
     <widget class="QPushButton" name="process_pb">
      <property name="text">
       <string>process</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="load_border_pb">
      <property name="text">
       <string>load border</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QPushButton" name="generate_partition_pb">
      <property name="text">
       <string>Generate partition</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QPushButton" name="load_partition_pb">
      <property name="text">
       <string>load partition</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>CCD-ID</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="CCD_ID_edit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="3" colspan="2">
     <widget class="QPushButton" name="two_partition_pb">
      <property name="text">
       <string>two-partition</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="two_load_pb">
      <property name="text">
       <string>two-load</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QPushButton" name="cutting_pb">
      <property name="text">
       <string>Cutting</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QPushButton" name="load_cutting_pb">
      <property name="text">
       <string>load cutting</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>mode</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="mode_edit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="training_pb">
      <property name="text">
       <string>training</string>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QPushButton" name="test_file_pb">
      <property name="text">
       <string>test file</string>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <widget class="QPushButton" name="load_threshold_pb">
      <property name="text">
       <string>load Threshold</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <spacer name="horizontalSpacer_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="3" colspan="2">
     <widget class="QPushButton" name="load_shade_pb">
      <property name="text">
       <string>load Shade</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>510</y>
     <width>521</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="left_move_pb">
      <property name="text">
       <string>left move</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="right_move_pb">
      <property name="text">
       <string>right move</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="up_move_pb">
      <property name="text">
       <string>up move</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="down_move_pb">
      <property name="text">
       <string>down move</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="test_pb">
      <property name="text">
       <string>test</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exit_pb">
      <property name="text">
       <string>exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>570</y>
     <width>411</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>project ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="project_id_edit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="export_modle_pb">
      <property name="text">
       <string>export modle</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="modle_test_pb">
      <property name="text">
       <string>modle test</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>311</width>
     <height>241</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="db_test_add">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>640</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>db_add</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
